src/scr_flush_sync.c

```diff
@@ -225,9 +231,6 @@ int scr_flush_sync(scr_cache_index* cindex, int id)
   /* free data structures */
   kvtree_delete(&file_list);
 
-  /* remove sync flushing marker from flush file */
-  scr_flush_file_location_unset(id, SCR_FLUSH_KEY_LOCATION_SYNC_FLUSHING);
-
```

Comment:
I think we want to leave this in.  Do you have to remove it for some reason?

Reply:
Good catch, that should be left in.  I think it's an artifact from the my old `XFER=SCR_POSTSTAGE` code.